.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - restore_cache:
```

Comment:
Do we need a step to explicitly install Yarn if it's not  already installed? The [React repo's `circle.yml` file](https://github.com/facebook/react/blob/master/circle.yml) has this:
```
  pre:
    # This is equivalent to $TRAVIS_COMMIT_RANGE
    - echo $CIRCLE_COMPARE_URL | cut -d/ -f7
    # install yarn if it's not already installed
    - |
      if [[ ! -e ~/.yarn/bin/yarn || $(yarn --version) != "${YARN_VERSION}" ]]; then
        curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
      fi
```

Maybe this isn't necessary with the 2.0 API. The 1.0 API docs have [a section about how to enable Yarn](https://circleci.com/docs/1.0/yarn/) which the 2.0 docs don't have. Maybe it just works out of the box now? :grin: Let's try and see!

Reply:
Pretty sure it’s already installed on v2

Reply:
Yeah, seems like that's the case.